index.html

```diff
@@ -3453,7 +3453,8 @@ <h2>Definition of Roles</h2>
 			<rdef>figure</rdef>
 			<div class="role-description">
 				<p>A perceivable <rref>section</rref> of content that typically contains a <a>graphical document</a>, images, code snippets, or example text. The parts of a <code>figure</code> MAY be user-navigable.</p>
-				<p>Authors SHOULD provide a reference to the <code>figure</code> from the main text, but the <code>figure</code> need not be displayed at the same location as the referencing element. Authors MAY reference text serving as a caption using <pref>aria-describedby</pref>. Authors MAY provide a label using <pref>aria-label</pref> or MAY reference text serving as a label using <pref>aria-labelledby</pref>.</p>
+				<p>Authors SHOULD provide a reference to the <code>figure</code> from the main text, but the <code>figure</code> need not be displayed at the same location as the referencing element. Authors MAY provide a <code>figure</code> a <rref>caption</rref> to serve as the name, description, or both to the element. Please refer to the <rref>caption</rref> role for more information on how to associate these roles.</p>
+				<p>If a <code>figure</code> does not contain a <code>caption</code>, authors MAY provide a <code>figure</code> an accessible name using <pref>aria-label</pref> or MAY reference other text serving as a label using <pref>aria-labelledby</pref>.</p>
```

Comment:
This sounds a little like authors can provide an `aria-label` if the figure doesn't have a caption (which wouldn't be true if the caption were a description, or neither a name nor a description).

I half-wonder if `caption` needs to be mentioned here at all -- the role doesn't make any difference to the naming/describing of figure, and it isn't called out for `table`, `grid`, or `treegrid`